### ISSUE TITLE
How to use three Trackball Control to Potree Viewer ?

### ISSUE THREAD

**Issue by EngIbrahimYoussef** [potree/potree#594]
hi,
Versions: Potree 1.6 / PotreeConverter_1.6_windows_64.
i have Clouds not plotted in true axis and i face problem in rotation with orbit control because it limited in up/down direction  with 180 is there is any way to disable limitation in orbit control or how can add Trackball control to potree viewer or replacing orbit control by trackball control

**Comment by chuxiaonan** [comment]
There's no state-of-art way to do what you want as I recall; however, you'd probably need to write a new navigation class and use it as default.

For example, you will create a new navigation control class named `TrackBallControls.js` in `src/navigation`, and replace `this.navigation = OrbitControls;` with `this.navigation = TrackBallControls;` in the `constructor` of `src/viewer/View.js`.

**Comment by EngIbrahimYoussef** [comment]
yes, there is no way to do that without   adding  TrackBallControls to potree viewer but will lead to many changes in src/viewer/View.js  because the trackballcontrols change in roll,pitch yaw and camera UP but till now all controls in potree change in pitch and yaw only
